<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Podrobno</class>
 <widget class="QWidget" name="Podrobno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>436</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>335</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Подробная информация</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>76</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Автор:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineAuthor"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>76</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineName"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_Number">
       <property name="minimumSize">
        <size>
         <width>76</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Номер:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineNumber"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Издательство:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineIzd"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>76</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Год издания:  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineData"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>76</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Отдел:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineOtdel"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>76</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Цена:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linePrice"/>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>76</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>76</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>Фонд:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="der">
       <item>
        <property name="text">
         <string>Художественная литература до 1917</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Художественная литература после 1917</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Зарубежная литература</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Детская литература</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Учебный фонд</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ценный фонд</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>В наличии: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="nal"/>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Содержание: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="content"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
